### ISSUE TITLE
Menu button - open a blank page

### ISSUE THREAD

**Issue by mickeyhoh** [Kikobeats/uno-zen#110]
![image](https://cloud.githubusercontent.com/assets/11407891/10445182/f2492194-71a1-11e5-93bc-f0014afde942.png)

When I opened the webpage within a browser (Google Chrome) on my notebook with a smaller screen, the red Menu button appears on the top left. When this button is clicked, a new but blank page is opened.
Is this a bug?

**Comment by Kikobeats** [comment]
what happens if you reload the browser?

**Comment by mickeyhoh** [comment]
The Red menu button will not appear if the screen is big, such as a full browser window on the notebook.

If you reduce the size of the browser to a size similar to the above (notice the scroll bar at the bottom of the picture), you will get that Red menu button. The button should appear on the top left of the white page (what do you call this HBS component?) and it should not appear on the top left of the Aside. Am I right?

Basically, when I click on the Red menu button, a new tab opens on the same browser but it is a blank page. In short, the Red button is not working as per intended.

I am not sure why you want to reload the browser but I reloaded the blog page and also the blank page. There is no change or any difference.

That said, the Red menu button works well onto a smartphone.. it just make the Aside appear and disappear but sliding it left to right and right to left.

In short, the problem is on the computer browser. I am using Chrome v45.

**Comment by foorb** [comment]
Yeah, I can confirm this. But when I reload the page it works.
![okt 13 2015 10 09](https://cloud.githubusercontent.com/assets/14993807/10449273/c90ceb26-7192-11e5-8bdc-d0ac0e2e14a6.gif)

**Comment by mickeyhoh** [comment]
Thanks for confirming @foorb.
When I refresh it, the Red menu appears where it belongs and it works fine.
The problem is, when you visit the site without reloading, the problem is there.

If a new visitor drops by, the Red menu button will appear wrongly.. he wouldn't know that he needs to refresh or reload.
So refreshing is not an elegant solution.. so the problem is still unresolved.

**Comment by Kikobeats** [comment]
Reload is necessary because media queries need to be detected to adapt the style. But is strange that you see a partial mobile view. maybe browser cache?

**Comment by mickeyhoh** [comment]
I was thinking about the cache as well. 

Since I have reloaded it just now, when I close my browser and visit your blog again, the Red button should have appear correctly. I expect this to happen since the cache is now has been updated.

However, this was not the case. The same thing happen. This means reloading only solve the current visit.. not the future visit.

@foorb, have you try clearing your cache?

**Comment by foorb** [comment]
I've done this with cleared cache.

**Comment by Kikobeats** [comment]
yeah! then feel free to close the issue :-)

**Comment by mickeyhoh** [comment]
I cleared cached.
Shrunk the browser to a smaller size, with similar size as my screen capture above.
Opened a new tab in Chrome.
Entered kikobeats.com as the URL.
Page loaded with the Red menu button correctly on the blog page (the aside didn't appear on the landing page).
Closed the same tab.
Opened a new tab in Chrome.
Entered kikobeats.com as the URL.
Page loaded with the Red menu button incorrectly on the aside (the aside appeared on the landing page), same screen as the screen capture above. Problem reappeared.
I cleared cached again.
Opened a new tab in Chrome.
Entered kikobeats.com as the URL.
Page loaded with the Red menu button correctly on the blog page (the aside didn't appear on the landing page).

This means that in order to load the page correctly, I need to clear my cache everytime I visit the blog... or reload the blog.
This approach is not the right solution.

**Comment by mickeyhoh** [comment]
I have tried opening up the blogs of adrianperez.org and also www.olddonkey.com (who just requested to add his blog onto your Github page). Theirs have no problem.
I am not sure if the recent fix for aside created this problem.

**Comment by foorb** [comment]
Yeah. It's a bit strange. I gone back to version 2.5.5- this version already have the problem.
But your right. The site adrianperez.org and olddonkey.com don't have the problem.

**Comment by mickeyhoh** [comment]
In that case, this is not as simple as a cache-clearing issue.
@Kikobeats I suspect this could be related to viewport?

**Comment by Kikobeats** [comment]
I think so

**Comment by mickeyhoh** [comment]
In that case, I will await your advice on the fix... thanks!

**Comment by NerosTie** [comment]
I have the same bug as described by @foorb in his gif.

**Comment by mickeyhoh** [comment]
Hi @Kikobeats. Is this still a bug?

**Comment by Kikobeats** [comment]
I'm think so. I need more time, or more people collaborating in the project.

**Comment by mickeyhoh** [comment]
Sure, no problem. I understand :)

**Comment by achreftlili** [comment]
i find out that this exist only with zepto but with jquery it's working
